sdk/resources/Microsoft.Azure.Management.Resource/src/Generated/Operations.cs

```diff
@@ -36,7 +36,7 @@ internal partial class Operations : IServiceOperations<ResourceManagementClient>
         /// <exception cref="System.ArgumentNullException">
         /// Thrown when a required parameter is null
         /// </exception>
-        internal Operations(ResourceManagementClient client)
+        internal Operations(SubscriptionClient client)
```

Comment:
These changes look like the source of your current error - you are generating multiple clients, each of which has its own Operations implementation, and you should be generating them into sub-directories.  This may mean that you need to update the Start-AutoRestCodeGeneration command to update the output directory for this.